README.md

```diff
@@ -1,12 +1,75 @@
 celvet
 ======
 
+**celvet** is a multitool for linting CRDs (`CustomResourceDefinition`) in
+Kubernetes. `celvet` also provides particular insight into [validation rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules),
+checking that they do not exceed the resource constraints provided by
+Kubernetes' API server. The tool is deliberately made to be useful both in an
+ad-hoc fashion by developers in addition to being part of a CI/CD pipeline, so
+`celvet` supports output in JSON in addition to machine-readable text.
+
+What does celvet catch?
+-----------------------
+
+* **CRD parsing errors** - celvet will early exit if it finds that a CRD
+    contains invalid YAML/JSON.
+* **Missing limits on variable-length fields** - `maxLength`, `maxItems`, etc.
+* **Validation rule parsing errors**  - if celvet has issues parsing a
+    validation rule, it will output the errors.
+* **Exceeding resource constraints for validation rules** - if a validation
+    rule exceeds the per-rule cost budget, celvet will output an error.
```

Comment:
Similar to above, clarify what a per-rule cost budget is?

Reply:
Gotcha, I've added a link to the relevant documentation here as well.